### PR TITLE
Items in the content lists on the coding standards page do nothing when clicked. #4

### LINKED ISSUE #4: Items in the content lists on the coding standards page do nothing when clicked.
When you click on a particular link in the content lists of either Java or HTML coding standards, the page does not scroll down to the correct section.

### PR DESCRIPTION
**Pull request by kowshik-sundararajan**
Fixed the issue by adding in id tags for the different headings. 
Fixes #4

### CODE REVIEW

**Comment by kowshik-sundararajan** [comment]
Noticed a typo in the markdown. In coding-standards-java.md, there is no clear 'Method' section. Thus I have not linked the content list item to the method section.

File: codingstandards/coding-standards-html.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 - <a href="#indentation">Indentation</a>
 - <a href="#miscellaneous">Miscellaneous</a>
```
Comment (damithc) [inline codingstandards/coding-standards-html.md]: @kowshik-sundararajan Does the `[Miscellaneous](#miscellaneous)` format work? That format doesn't require adding anchors to the main text of the document.

**Comment by kowshik-sundararajan** [comment]
Hi Prof @damithc , that's a great suggestion! While it works for the ToC in `coding-standards-html.md` because the list is in pure Markdown, I have not been able to make it work for the ToC in `coding-standards-java.md` because the html tags wrap the list. Please advise.

**Comment by damithc** [comment]
I see. Shall we remove the html tags around Java ToC (and also get rid of Ada's image)? The image is not worth the loss of markdown convenience.

**Comment by kowshik-sundararajan** [comment]
Sure, I have updated the `md` files.

**Comment by damithc** [comment]
If the image file is no longer used anywhere else, you can delete it as well.

**Comment by damithc** [comment]
Thanks @kowshik-sundararajan 
It works for the markdown version but not for the [html version](https://oss-generic.github.io/process/codingstandards/coding-standards-java.html). Probably our markdown-to-html conversion script doesn't auto-generate anchors for headings.

**Comment by damithc** [comment]
@kowshik-sundararajan you can give #1 a try if you are up for it. The fix is to add html anchors for headings manually.

**Pull request merged**